.Rproj.user/84277BCC/cpp-complilation-config

```diff
@@ -7,7 +7,7 @@
         "-IE:/R-3.6.3/include",
         "-DNDEBUG"
     ],
-    "hash" : "1582475403",
+    "hash" : "1587558325",
```

Comment:
@z267xu I think we should probably `.gitignore` the `.Rproj.user` directory altogether 🙂

Reply:
Sure. It is now